setup.py

```diff
@@ -33,10 +33,6 @@
         'console_scripts': [
             'run-ci=run_ocsci:main',
         ],
-        'pytest11': [
-            'ocscilib = ocsci.pytest_customization.ocscilib',
-            'marks = ocsci.pytest_customization.marks',
-        ]
```

Comment:
How did you register these modules for normal test run (not unittests)?

Reply:
I'm ok with dropping this from setup, but we need to add it into global conftest (which then would need to be shielded from unittests) or in other appropriate place.

Reply:
See https://github.com/red-hat-storage/ocs-ci/pull/165/files#diff-52ce28eee4cc4cda242e445bb4ac28d1R113

Reply:
This should be resolved I guess